fastlane/Fastfile

```diff
@@ -67,7 +67,9 @@ ENV["HAS_ALPHA_VERSION"]="true"
     setbranchprotection(repository:GHHELPER_REPO, branch: "release/#{new_version}")
     setfrozentag(repository:GHHELPER_REPO, milestone: new_version)
     
+    localize_gutenberg(skip_module_update: true)
```

Comment:
Just a note to say that's great we're calling `gutenberg_dep_check()`  before that, that should ensure the bundle, and bundled strings are up to date.